acceptance/suites/tests/service_framework/reload_updated_crl.rb

```diff
@@ -78,24 +78,9 @@
   on(master, puppet('cert', 'revoke', cert_to_revoke))
 end
 
-if options[:type] == 'pe'
-  step 'Copy cacrl to hostcrl in order to work around SERVER-911' do
-    on(master, 'cp "$(puppet config print cacrl)" "$(puppet config print hostcrl)"')
-  end
-end
-
-step 'Validate that noop agent run successful after cert revoked but before reload' do
-  on(master, puppet('agent', '--test',
-                    '--server', server,
-                    '--certname', cert_to_revoke,
-                    '--noop'))
-end
-
-step 'Reload the server' do
-  reload_server
-end
-
-step 'Validate that noop run for revoked agent fails with SSL error after server reload' do
+step 'Validate that noop run for revoked agent fails with SSL error after CRL reload' do
+  # Sleep for a few seconds to give the server a chance to reload the updated CRL
+  sleep 5
```

Comment:
This may not strictly be necessary.  The test passed fine for me without it.  Just seemed like this would make the test a bit less likely to fail.